Splunk_TA_paloalto/default/transforms.conf

```diff
@@ -65,6 +65,11 @@ DEST_KEY = MetaData:Sourcetype
 REGEX = ^[^,]+? - config,
 FORMAT = sourcetype::pan:config_traps
 
+[pan_decryption]
+DEST_KEY = MetaData:Sourcetype
+REGEX = ^[^,]+,[^,]+,[^,]+,DECRYPTION,
+FORMAT = sourcetype::pan:decryption
+
```

Comment:
Since this is a PAN-OS log type, lets keep all the PAN-OS log types together. So maybe slot this in between `pan_globalprotect` and `pan_traps4`.